### PR TITLE
[plugin.video.pcloud-video-streaming] 1.2.5

### PR DESCRIPTION
**Pull request by gdomenici**
Solves issue reported at http://forum.kodi.tv/showthread.php?tid=239955&pid=2536802#pid2536802

### CODE REVIEW

File: plugin.video.pcloud-video-streaming/resources/language/Italian/strings.xml [hunk 1]
```diff
@@ -22,4 +22,6 @@
   <string id="30119">Cancellazione fallita (vedi log)</string>
```
Comment (enen92) [inline plugin.video.pcloud-video-streaming/resources/language/Italian/strings.xml]: please consider to move to the new language format (.po files in the future)

File: plugin.video.pcloud-video-streaming/addon.xml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<addon id="plugin.video.pcloud-video-streaming" name="PCloud Video Streaming" version="1.2.3" provider-name="Guido Domenici">
+<addon id="plugin.video.pcloud-video-streaming" name="PCloud Video Streaming" version="1.2.5" provider-name="Guido Domenici">
   <requires>
     <import addon="xbmc.python" version="2.1.0"/>
```
Comment (enen92) [inline plugin.video.pcloud-video-streaming/addon.xml]: please consider to set this one to 2.19.0 in the future (so it matches helix). See: http://kodi.wiki/view/addon.xml

**Comment by enen92** [comment]
Please rename the PR to "[plugin.video.pcloud-video-streaming] 1.2.5" so we can merge it

**Comment by gdomenici** [comment]
Hi @enen92 ,

> Please rename the PR to "[plugin.video.pcloud-video-streaming] 1.2.5" so we can merge it

Done, thanks. Will implement the rest of the suggestion with the next update.

**Pull request merged**